recipe_price_exercise.py

```diff
@@ -0,0 +1,23 @@
+def get_recipe_price(prices, optionals=[], **ingredients):
+    # Check if input is correct
```

Comment:
I would use documentation instead, comments make it look messy
https://swimm.io/blog/code-comments-vs-documentation/

Reply:
fixed it